site/docs/src/json/two-factor/start/request.json

```diff
@@ -4,5 +4,6 @@
   "code": "123456",
   "state": {
     "redirect_uri": "https://fusionauth.io"
-  }
+  },
+  "two-factor-trust": "<UUID_from_two/factor/login>"
```

Comment:
I don't think we have any precedence in docs for making values like this. I would suggest we use a value that is representative of the actual value. If we need to describe the value further - it should be in the API request parameter description.